scripts/deployment/deploy_daily_rewards.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
```

Comment:
A lot of code duplication. Maybe a semi-generic contract deployer in `common.js`?

Reply:
not sure about that.
different validations, different arguments to constructor, and staking using 'deployProxy'.